linux/scripts/deb-packaging.sh

```diff
@@ -168,7 +176,12 @@ is_api_version_updated() {
     alpha)
       local STABLE_VERSION
       STABLE_VERSION=$((${VERSION%%.*} - 1))
-      git checkout "stable-${STABLE_VERSION}.0" -- "debian/${PKG_NAME}.symbols"
+      if ! git cat-file -e "origin/stable-${STABLE_VERSION}.0" "debian/${PKG_NAME}.symbols" 2>/dev/null; then
```

Comment:
This command looks malformed?

```suggestion
      if ! git cat-file -e "origin/stable-${STABLE_VERSION}.0:debian/${PKG_NAME}.symbols" 2>/dev/null; then
```

Reply:
Good catch. Thanks!